infinispan-spring-boot-starter-embedded/src/main/java/org/infinispan/spring/starter/embedded/InfinispanEmbeddedAutoConfiguration.java

```diff
@@ -60,17 +61,20 @@
    public DefaultCacheManager defaultCacheManager() throws IOException {
       final String configXml = infinispanProperties.getConfigXml();
       final DefaultCacheManager manager;
+      GlobalConfigurationBuilder globalConfigurationBuilder = new GlobalConfigurationBuilder();
+      globalConfigurationBuilder.serialization().marshaller(new JavaSerializationMarshaller());
+      globalConfigurationBuilder.serialization().whiteList().addClass("org.springframework.session.MapSession");
+      globalConfigurationBuilder.serialization().whiteList().addRegexp("java.util.*");
```

Comment:
A bit too wide for my taste, if other implementations also allow all of `java.util` by default it would be nice to have a comment in the source code.

Reply:
yes, we need it here for this case. I will add a comment